pages/en/contrib/code-contrib.md

```diff
@@ -8,96 +8,5 @@ sidebar: contrib_sidebar
 permalink: /doc/en/contrib/code-contrib.html
 ---
 
-## How to contribute to the code
-
-You can contribute in a number of ways:
-
-* [Report an issue](#reporting-an-issue).   Reporting it first will ensure that it is
-  in fact an issue, and there is not already a fix in the works.
-* After reporting a bug, go one step further and fix it.
-* Ensure that your effort is aligned with the project's roadmap by
-  talking to the maintainers, especially if you are going to spend a
-  lot of time on it.
-* Adhere to code style outlined in the [style guide](style-guide.html).
-* Agree to the [Contributor License Agreement](#agreeing-to-the-cla).
-* [Submit a pull request](#submitting-a-pull-request) through Github.
-* Run tests; see [Running tests](#running-tests).
-
-## Where to start
-
-If you are looking to contribute, but aren't sure where to start, review open pull requests for the module.  For example: [LoopBack open pull requests](https://github.com/strongloop/loopback/pulls).
-
-Also, please read the source code before contributing. If something is confusing, poorly documented, poorly commented, or could otherwise be improved: open a GitHub issue and link to the file and line of code with a useful description of what you would like to see or think is missing.
-
-Another place to look is to find our issues on GitHub with `good first issue` tag.  See 
```

Comment:
The section "Where to start" is applicable to both 3.x and 4.x, let's keep it here please.

Just change the links to open pull requests & good first issues - offer two links, one for `strongloop/loopback` and the other for `strongloop/loopback-next`.

Reply:
I've added the links of PRs for all versions.  For the good first issues link, I kept it unchanged because the link is for all the repos in the `strongloop` org. thanks.